doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -201,7 +201,10 @@ nRF5340 Audio
 nRF Machine Learning (Edge Impulse)
 -----------------------------------
 
-|no_changes_yet_note|
+* Updated the MCUboot and ``hci_rpmsg`` child image debug configurations to disable the :kconfig:option:`CONFIG_RESET_ON_FATAL_ERROR` Kconfig option.
```

Comment:
`MCUboot and HCI RMPsg child images` -> to improve consistency
Also in other places